moped-editor/src/layouts/DashboardLayout/TopBar.js

```diff
@@ -106,7 +106,7 @@ const TopBar = ({ className, onOpen, ...rest }) => {
         </RouterLink>
         <Hidden smDown>
           <Box>
-            <Tabs className={classes.tabs}>
+            <Tabs className={classes.tabs} value={false}>
```

Comment:
we're not using the `value` prop to set tab state, but it needs a value regardless. `undefined` throws an error.